src/lib/accessibility/Behaviors/Popup/PopupBehavior.ts

```diff
@@ -0,0 +1,64 @@
+import { Accessibility } from '../../interfaces'
+import * as keyboardKey from 'keyboard-key'
+import _ from 'lodash'
+
+/**
+ * @description
+ *  Adds role='button' to 'trigger' component's part, if it is not focusable element and no role attribute provided.
+ *  Adds tabIndex='0' to 'trigger' component's part, if it is not tabbable element and no tabIndex attribute provided.
+ *  Adds attribute 'aria-disabled=true' to 'trigger' component's part based on the property 'disabled'.
+ *  Adds attribute 'aria-haspopup=true' to 'trigger' component's part.
+ */
+const PopupBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    trigger: {
+      role: getAriaAttribute('role', props, 'button'),
+      tabIndex: getAriaAttribute('tabIndex', props, '0'),
+      'aria-haspopup': 'true',
+      'aria-disabled': !!props['disabled'],
+    },
+  },
+  keyActions: {
+    trigger: {
+      performClick: {
+        keyCombinations: [{ keyCode: keyboardKey.Enter }, { keyCode: keyboardKey.Spacebar }],
```

Comment:
What's intended behavior here?

I understand I can open popup using Enter or Spacebar - where should the focus go after that? Now it remains on the trigger, there is no way how I could navigate inside the popup. What if there is a menu inside the popup?

And when the focus remains on the trigger, I would expect it would be closable by Esc (copying `closeAndFocusTrigger` from ~5 lines below to here works for me), and personally I am not sure that I expect Enter to close it.

Reply:
Focus management would be introduced in future PR, this PR just adds accessibility behavior + action handlers.

Regarding closing the popup by Esc, before it was implemented as you mentioned. But after discussion with Milan, we came up that it is not correct from an accessibility point of view. Esc should work for a popup to close it when the focus is on the popup. For trigger element, it will behave the same way as clicking with a mouse - which will toggle popup currently.